<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JosephusCircleFrame</class>
 <widget class="QFrame" name="JosephusCircleFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>221</width>
     <height>180</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_people">
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>人数</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_times">
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>报数周期</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_starter">
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>起始人</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="spinBox_people">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="spinBox_times">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="spinBox_starter">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QPushButton" name="commitButton">
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>提交</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBox">
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>数组</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>链表</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="labelDataStructure">
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>选择数据结构</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>361</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTextBrowser" name="labelResult">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>256</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
